astro/src/content/docs/apis/_user-search-request-parameters.mdx

```diff
@@ -11,21 +11,20 @@ import SearchPreprocessingWarning from "src/content/docs/_shared/_search-preproc
 
 <APIBlock>
 
-
-  <APIField name={props.parameter_prefix + 'accurateTotal'} type="Boolean" optional defaults="false" since="1.24.0">
-    Set this value equal to `true` to receive an accurate hit count on the API response.
-
-    By default the search engine will limit the hit count to 10,000 users. This means that even if your query may match more than 10,000 users, the returned total count will be 10,000. This is adequate for many use cases such as pagination and general purpose queries. If you are looking for an accurate user count that can exceed 10,000 matches, you will want to set this value equal to `true`.
-  </APIField>
-
-  <APIField name={props.parameter_prefix + 'expand'} type="Array<String>" optional defaults="[memberships, registrations]" since="1.48.0">
+  <APIField name='expand' type="Array<String>" optional defaults="[memberships, registrations]" since="1.48.0">
```

Comment:
```suggestion
  <APIField name="expand" type="Array<String>" optional defaults="[memberships, registrations]" since="1.48.0">
```

Fix quoting on expand.